neurodocker/cli/minify/tests/test_minify.py

```diff
@@ -8,9 +8,11 @@
 docker = pytest.importorskip("docker", reason="docker-py not found")
 
 
+@pytest.mark.skip(reason="ptrace no longer supported under docker")
 def test_minify():
     client = docker.from_env()
-    container = client.containers.run("python:3.9-slim", detach=True, tty=True)
+    container = client.containers.run("python:3.9-slim", detach=True, tty=True,
+                                      platform="Linux/amd64", privileged=True)
```

Comment:
i wonder if adding `cap_add=["SYS_PTRACE"]`  would fix this. not a priority for now but hopefully we can get this working again. though I wonder if `privileged=True` implies that.

Reply:
i have tried all variants, none of them worked on my m1 mac.

Reply:
and from docker's docs it seems they may have fully disabled this.